gradio/components.py

```diff
@@ -652,6 +655,26 @@ def __init__(
             **kwargs,
         )
 
+    @staticmethod
+    def round_to_precision(num: float | int | None, precision: int | None):
+        """Round to a given precision.
```

Comment:
Let's follow the codebase
```suggestion
        """
        Round to a given precision.
```